driver/razerkbd_driver.c

```diff
@@ -1011,6 +1012,10 @@ static ssize_t razer_attr_read_device_type(struct device *dev, struct device_att
         device_type = "Razer Blade 17 (2022)\n";
         break;
 
+    case USB_DEVICE_ID_RAZER_BLADE_14_2022:
+        device_type = "Razer Blade 14 (2022)\n";
```

Comment:
Do they do the "early" or "mid" or "late" designations for the 14s? If not, this is totally fine.